test/acceptance/test/add_tests.go

```diff
@@ -658,12 +637,26 @@ var _ = Describe("Weave GitOps Add App Tests", func() {
 			deleteWorkload(tip.workloadName, tip.workloadNamespace)
 		})
 
-		By("When I create an empty private repo for app", func() {
+		By("And I create a private repo with my app workload", func() {
 			repoAbsolutePath = initAndCreateEmptyRepo(appRepoName, gitProvider, true, gitOrg)
+			gitAddCommitPush(repoAbsolutePath, tip.appManifestFilePath)
 		})
 
-		By("And I git add-commit-push for app", func() {
-			gitAddCommitPush(repoAbsolutePath, tip.appManifestFilePath)
+		By("And Gitops runtime is not installed in Cluster2", func() {
+			selectCluster(cluster2Context)
+			uninstallWegoRuntime(WEGO_DEFAULT_NAMESPACE)
+		})
+
+		By("And gitops check pre kubernetes version is compatible and flux is not installed in Cluster2", func() {
+			selectCluster(cluster2Context)
+			c := exec.Command(gitopsBinaryPath, "check", "--pre")
+			output, err := c.CombinedOutput()
+			Expect(err).NotTo(HaveOccurred())
+			expectedOutput := fmt.Sprintf(`✔ Kubernetes %s.* >=[0-9]+.[0-9]+\.[0-9]+-[0-9]+
```

Comment:
I wonder if there is any particular reason for `.*` after `%s`. I ask because there is no `.*` in the test `And gitops check pre validates kubernetes and flux compatibility for Cluster1` where check pre is called as well.

Reply:
good catch on that - i need to get that removed